src/main/java/com/mule/einstein/internal/helpers/documents/ParametersEmbeddingDocument.java

```diff
@@ -15,16 +16,21 @@ public class ParametersEmbeddingDocument {
   @Expression(ExpressionSupport.SUPPORTED)
   @OfValues(EmbeddingModelApiNameProvider.class)
   @Optional(defaultValue = OPENAI_ADA_002)
```

Comment:
@sf-vishwastyagi should we chane this also to be api name, in-line with dropdown values?

Reply:
I would say, modelApiName is better

Reply:
I have updated PR